core/src/trezor/ui/layouts/tt.py

```diff
@@ -389,13 +390,43 @@ def confirm_replacement(
     return interact(ctx, Confirm(text), "confirm_replacement", ButtonRequestType.SignTx)
 
 
+def confirm_modify_output(
+    ctx: wire.GenericContext,
+    address: str,
+    sign: int,
+    amount_change: str,
+    amount_new: str,
+) -> LayoutType:
+    page1 = Text("Modify amount", ui.ICON_SEND, ui.GREEN)
+    page1.normal("Address:")
+    page1.br_half()
+    page1.mono(*_split_address(address))
+
+    page2 = Text("Modify amount", ui.ICON_SEND, ui.GREEN)
+    if sign < 0:
+        page2.normal("Decrease amount by:")
+    else:
+        page2.normal("Increase amount by:")
```

Comment:
Note that this branch is unreachable, so as an alternative  we could raise an exception here or rename the function to `confirm_decrease_output()`.

Reply:
i'd keep this as-is, this is rather far away from the code that knows about the unreachability